hopo/studiengebuehren.tex

```diff
@@ -2,23 +2,25 @@
+Weiterhin müsst ihr jedoch als Teil der \EUR{\beitragssumme}, die ihr jedes Semester an die Uni überweisen müsst, einen "Verwaltungskostenbeitrag" von \EUR{\verwaltungsbetrag} bezahlen. Dieser wird den Hochschulen auf ihren Etat angerechnet, sodass das Geld de facto als Ersparnis ans Land Baden-Württemberg geht. Weil das Land nach der nächsten Wahl die allgemeinen Studiengebühren als Mittel gegen die erste Anhebung des Hochschuletats seit 1996 wieder entdecken könnte, möchten wir das Thema Studiengebühren nicht einfach als Teil der hochschulpolitischen Geschichte abhaken, sondern in diesem Kapitel daran erinnern. Wir laden Euch darüber hinaus ein, Euch über die Hintergründe der Studiengebühren zu informieren, beispielsweise auf unserer Fachschaftshomepage \footnote{\url{https://mathphys.fsk.uni-heidelberg.de/w/hintergruende/historie-der-studiengebuehren}}.
 
-\subsection*{Ein paar Argumente gegen Studiengebühren}
+\subsection*{Ein paar Argumente gegen allgemeine Studiengebühren}
 
 Wie ihr sicher bemerkt habt, gehen wir davon aus, dass es gut sei, wenn es keine Studiengebühren gibt. Was veranlasst uns dazu, mal abgesehen von persönlicher Betroffenheit?
 \begin{itemize}
 \item {Barrieren zwischen Abi und Studium einreißen!}\\
-	Ein Studium ist verglichen mit einer Ausbildung sehr teuer. Mit Studiengebühren wäre es nochmals knapp \EUR{100}/Monat teurer. Das schreckt nachweislich insbesondere AbiturientInnen ab, deren Eltern selbst nicht studiert haben. Außerdem schreckt es prozentual mehr Mädchen als Jungs ab. Kurzum: Menschen, deren Umgebung von ihnen traditionell weniger erwartet dass sie studieren, studieren weniger; gesellschaftliche Ungleichheit, Stratifizierung und überkommene Rollenbilder werden fortgeschrieben.
+	Ein Studium ist verglichen mit einer Ausbildung sehr teuer. Mit allgemeinen Studiengebühren wäre es nochmals knapp \EUR{100}/Monat teurer. Das schreckt nachweislich insbesondere AbiturientInnen ab, deren Eltern selbst nicht studiert haben. Außerdem schreckt es prozentual mehr Mädchen als Jungs ab. Kurzum: Menschen, deren Umgebung von ihnen traditionell weniger erwartet dass sie studieren, studieren weniger; gesellschaftliche Ungleichheit, Stratifizierung und überkommene Rollenbilder werden fortgeschrieben.
```

Comment:
Abiturientinnen in geänderten Texten kann ruhig gleich wie beschlossen gegendert werden niemand muss warten bis #21 abgearbeitet ist :stuck_out_tongue: